### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jaanonim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
 `
 py -m PyInstaller SyncDirectories.spec --noconfirm
 `
+
+## Authors:
+- [jaanonim](https://github.com/jaanonim)
+- [n2one](https://github.com/n2oneProgrammer)
+
+## Icon:
+Made by [mwawrzen](https://github.com/mwawrzen).
```
Comment (n2oneProgrammer) [inline README.md:15]: Idk if its a good idea if we will start in mfi

**Review by n2oneProgrammer — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (jaanonim) [reply README.md:15]: me too
but it will be unfair if we not mention it

Comment (n2oneProgrammer) [reply README.md:15]: ok marge it, becouse in rules , I didnt find it is inlegal

**Pull request merged**